jena/src/main/scala/org/w3/banana/jena/io/JenaRDFReader.scala

```diff
@@ -62,10 +62,22 @@ object JenaRDFReader {
     }
   }
 
+  def makeAnyRDFReader = new AnyRDFReader[Jena, Try] {
```

Comment:
Make it a real function, add `()`. And make sure you call it right in the module.